src/components/Image/Image.tsx

```diff
@@ -0,0 +1,35 @@
+import { HTMLAttributeAnchorTarget } from "react"
+
+export interface ImageProps {
+  src?: string
+  alt?: string
+  width?: number
+  href?: string
+  openInNewTab?: boolean
+}
+
+const Image = ({ src, alt, width }: ImageProps) => (
+  <img src={src} alt={alt} width={`${width ?? 100}%`} />
+)
+
+const ImageComponent = ({
+  src,
+  alt,
+  width,
+  href,
+  openInNewTab,
+}: ImageProps) => {
+  return (
+    <div>
+      {href ? (
+        <a href={href} target={openInNewTab ? "_blank" : undefined}>
```

Comment:
suggestion: can add `rel="noopener noreferrer nofollow"` for external links for security.

Reply:
[72d1921](https://github.com/isomerpages/isomer-components/pull/7/commits/72d19211abe7a233909a00117f941a55718a4fd0)

great suggestion, thanks!